numbat/src/value.rs

```diff
@@ -85,6 +86,7 @@ impl std::fmt::Display for Value {
             Value::String(s) => write!(f, "\"{}\"", s),
             Value::DateTime(dt) => write!(f, "datetime(\"{}\")", dt),
             Value::FunctionReference(r) => write!(f, "{}", r),
+            Value::FormatSpecifiers(_) => unreachable!(),
```

Comment:
Are you sure this isn't called anywhere? Maybe it's best just to implement it... e.g. for debugging purposes
```suggestion
            Value::FormatSpecifiers(_) => write(f, "<format specifier>"),
```